### DISCUSSION TITLE
Is there a way to export a single backslash from norg to markdown?

### DISCUSSION THREAD

**Discussion by 0x546F6D** [nvim-neorg/neorg#1148] [Q&A]
Hi,
First, I should probably mention that I dont know much about markdown nor LaTeX, and I am just getting into neorg. Hence the following is just the result of a few google searches:
I tried to understand how to convert my .norg notes to pdf and the common given method seems to be through markdown.

So I added core.export/core.export.markdown to my config, installed md-pdf.nvim plugin, and it works fine.
I just wanted to set some page breaks here and there in the final pdf, and so discovered the LaTeX command "\pagebreak", which you can write inside markdown files. That works fine too.

My only issue now is how to write those "\pagebreak" directly into my norg notes:
- if I write "\pagebreak" in note.norg, it becomes "pagebreak" in note.md
- if I write "\\\pagebreak" in note.norg, it stays as "\\\pagebreak" in note.md

So I currently write "\\\pagebreak" in my note.norg, do a search/replace in the exported note.md, and then export that to pdf.

So:
- Is there a way to export a single backslash from norg to markdown?
- Is there another solution to generate pdfs with page breaks from .norg files?

Thanks for your suggestions

**Comment by 0x546F6D** [comment]
my current dirty workaround:

``` lua
-- Replace \\pagebreak to \pagebreak for markdown files
vim.api.nvim_create_autocmd("FileType", {
  group = vim.api.nvim_create_augroup("markdown_replace", { clear = true }),
  pattern = { "markdown" },
  callback = function()
    vim.api.nvim_command("silent! %s/\\\\\\\\pagebreak/\\\\pagebreak/")
    vim.api.nvim_command("silent w")
  end,
})
```

**Comment by pysan3** [comment]
Use this and through pandoc you should be able to generate pdf with `-t pdf` without any complicated methods. 

You might want to try out the norganic section as well if it doesn't work as expected. 

https://github.com/pysan3/Norg-Tutorial/blob/main/norg_tutorial.md#export-norg-pandoc
